semantic_ai/connectors/microsoft/sharepoint.py

```diff
@@ -54,7 +66,7 @@ async def __make_request(self, url):
             'Authorization': 'Bearer {}'.format(await self.__get_access_token())
         }
         async with httpx.AsyncClient() as cli:
-            r = await cli.get(url, headers=headers)
+            r = await cli.get(url, headers=headers, timeout=60)
```

Comment:
By default, this would be `30`. `60` is more.

Reply:
changed